apps/token/.env.capsule

```diff
@@ -1,5 +1,5 @@
 # App configuration variables
-NX_VEGA_ENV=TESTNET
+NX_VEGA_ENV=CUSTOM
```

Comment:
This should be changed in the token-e2e project

Reply:
Somehow there's no problem with e2e, but when I built just token app locally

Reply:
We only need the e2e project to build to capsule for the tests. Token should build against testnet by default. Or it might be stagnet 3 soon but it should not be capsule.